packages/react/src/templates/FeaturePreviewLPs/FeaturePreviewLevelOne/FeaturePreviewLevelOne.tsx

```diff
@@ -0,0 +1,254 @@
+
+  const handleMode = e => {
+    e.preventDefault()
+    setIsLightMode(!isLightMode)
+  }
+
+  return (
+    <ThemeProvider
+      colorMode={isLightMode ? 'light' : 'dark'}
+      style={{
+        ['--brand-Pillar-icon-color-default' as string]: accentColorValue,
+        ['--brand-Label-color-default' as string]: accentColorValue,
+        ['--brand-color-accent-primary' as string]: accentColorValue,
+        backgroundColor: 'var(--brand-color-canvas-default)',
+      }}
+    >
+      <SubdomainNavBar title="Preview" fixed={false}>
+        <SubdomainNavBar.SecondaryAction href="#" onClick={handleMode}>
+          Change color mode
+        </SubdomainNavBar.SecondaryAction>
+        <SubdomainNavBar.PrimaryAction href="#" onClick={handleOverlay}>
+          {enableGridOverlay ? 'Disable' : 'Enable'} grid
+        </SubdomainNavBar.PrimaryAction>
+      </SubdomainNavBar>
+      <div>
+        <Box backgroundColor="subtle" marginBlockEnd={96} className={styles['FeaturePreview__trailingSection']}>
+          <Grid enableOverlay={enableGridOverlay}>
+            <Grid.Column>
+              <Hero align="start">
+                <Hero.Label>Release type</Hero.Label>
+                <Hero.Heading>Get this sweet benefit with [feature name]</Hero.Heading>
+                <Hero.Description>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus
+                  sed turpis felis nam pulvinar.
+                </Hero.Description>
+                <Hero.PrimaryAction href="#">Primary CTA</Hero.PrimaryAction>
+              </Hero>
+            </Grid.Column>
+          </Grid>
+        </Box>
+        <Grid enableOverlay={enableGridOverlay}>
+          <Grid.Column>
+            <section>
+              <Grid enableOverlay={enableGridOverlay}>
+                <Grid.Column span={5}>
+                  <Box marginBlockEnd={96}>
+                    <Stack padding="none">
+                      <Heading as="h2" size="5">
+                        Signup for this cool feature to enable you to do this before everyone else.
+                      </Heading>
+                      <Stack direction="vertical" padding="none" alignItems="flex-start">
+                        <Text as="p" variant="muted">
+                          It;s often considered polit to tell people what they are about to sign up for. Even if it
+                          involves making them read a bunch. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In
+                          sapien sit ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum.
+                        </Text>
+                        <Link variant="accent" href="#">
+                          Learn more about this
+                        </Link>
+                      </Stack>
+                    </Stack>
+                  </Box>
+                </Grid.Column>
+                <Grid.Column span={6} start={7}>
```

Comment:
We should stack this one vertically on smaller viewports

<img width="342" alt="image" src="https://github.com/primer/brand/assets/175638/004cd77b-4004-4eed-b1b6-6e5d953a8b48">